sdk/service/codec.go

```diff
@@ -2,8 +2,10 @@ package servicesdk
 
 import (
 	"github.com/mesg-foundation/engine/codec"
+	"github.com/mesg-foundation/engine/service"
 )
 
 func init() {
 	codec.RegisterConcrete(msgCreateService{}, "service/create", nil)
+	codec.RegisterConcrete(&service.Service{}, "mesg.types.Service", nil)
```

Comment:
why do you need to register service to the codec? I tried without and its working well.
Only struct used inside an interface should be registered. The msgs have to be register as they are used in Tx as interface